### PR TITLE
Add books service and integrate it with sign up view

### PR DESCRIPTION
**Pull request by 9gustin**
## Summary

Add service to do sign up and login with the API
Integrate service to sign up view

## Screenshots

image

## Trello Card

[Sign up II - Integration](https://trello.com/c/WbgCzaTO/119-sign-up-ii-integration)

### CODE REVIEW

File: react/w-books/.env.development [hunk 1]
```diff
@@ -0,0 +1 @@
+REACT_APP_API_BASE_URL=https://books-training-rails.herokuapp.com/api/v1/
```
Comment (DavidRojas1612) [inline react/w-books/.env.development:1]: Add this file to .gitignore file and then run **git rm --cached .env.development**

File: react/w-books/src/app/components/Loading/index.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      [styles.isGreen]: isGreen
+    }
+    )}>
+      <g>
+        <path className={styles.iconPath} d="M75.4 126.63a11.43 11.43 0 0 1-2.1-22.65 40.9 40.9 0 0 0 30.5-30.6 11.4 11.4 0 1 1 22.27 4.87h.02a63.77 63.77 0 0 1-47.8 48.05v-.02a11.38 11.38 0 0 1-2.93.37z" fill="#000000" fill-opacity="1" />
+        <animateTransform attributeName="transform" type="rotate" from="0 64 64" to="360 64 64" dur="1000ms" repeatCount="indefinite"></animateTransform>
+      </g>
+    </svg>
+  )
+}
```
Comment (DavidRojas1612) [inline react/w-books/src/app/components/Loading/index.tsx:23]: add eof

File: react/w-books/src/app/components/Routes/paths.ts [hunk 3]
```diff
@@ -1,6 +1,7 @@
 const PATHS = {
   login: '/',
-  signUp: '/sign_up'
+  signUp: '/sign_up',
+  home: '/home'
 };
 
 export default PATHS;
```
Comment (DavidRojas1612) [inline react/w-books/src/app/components/Routes/paths.ts:7]: add eof

File: react/w-books/src/config/api.ts [hunk 4]
```diff
@@ -3,6 +3,7 @@ import { CamelcaseSerializer, SnakecaseSerializer } from 'cerealizr';
 const deserializer = new CamelcaseSerializer();
 const serializer = new SnakecaseSerializer();
 const baseURL = process.env.REACT_APP_API_BASE_URL;
+console.log(baseURL);
```
Comment (DavidRojas1612) [inline react/w-books/src/config/api.ts:6]: delete this

File: react/w-books/src/services/userService.ts [hunk 5]
```diff
@@ -0,0 +1,9 @@
+import { ApiResponse } from 'apisauce';
+import api from '../config/api';
+
+export const SIGN_UP = '';
+export const LOGIN = '/sign_in';
+
+export async function postUser(action: typeof SIGN_UP | typeof LOGIN, user:any):Promise<ApiResponse<any, any>> {
+  return api.post(`users${action}`, user);
+}
```
Comment (DavidRojas1612) [inline react/w-books/src/services/userService.ts:9]: add eof

**Review by DavidRojas1612 — APPROVED** [review]

**Review by DavidRojas1612 — CHANGES_REQUESTED** [review]
a little error

File: react/w-books/src/services/userService.ts (hunk 5 above)
Comment (9gustin) [reply react/w-books/src/services/userService.ts]: ready

File: react/w-books/src/config/api.ts (hunk 4 above)
Comment (9gustin) [reply react/w-books/src/config/api.ts]: ready

File: react/w-books/src/app/components/Routes/paths.ts (hunk 3 above)
Comment (9gustin) [reply react/w-books/src/app/components/Routes/paths.ts]: ready

File: react/w-books/src/app/components/Loading/index.tsx (hunk 2 above)
Comment (9gustin) [reply react/w-books/src/app/components/Loading/index.tsx]: ready

File: react/w-books/.env.development (hunk 1 above)
Comment (9gustin) [reply react/w-books/.env.development]: ready

**Review by DavidRojas1612 — APPROVED** [review]

**Pull request merged**